doc/cookbook/source/examples/regression/least_squares_regression.rst

```diff
@@ -0,0 +1,53 @@
+========================
+Least Squares Regression
+========================
+
+A Linear regression model can be defined as :math:`y_i = \bf{w}^\top\bf{x_i}` where :math:`y_i` is the predicted value, :math:`\bf{x_i}` is a feature vector and :math:`\bf{w}` is the weight vector. We aim to find the linear function that best explains the data, i.e. minimizes the error or loss function :math:`E(\bf{w})` by finding appropriate :math:`\bf{w}`.
+
+.. math::
+    E({\bf{w}}) = \sum_{i=1}^N(y_i-{\bf w}^\top {\bf x}_i)^2
+
+One can differentiate :math:`E(\bf{w})` with respect to :math:`\bf{w}` and equate to zero to determine the :math:`\bf{w}` that minimizes :math:`E(\bf{w})`. This leads to solution of the form:
+
+.. math::
+    {\bf w} = \left(\sum_{i=1}^N{\bf x}_i{\bf x}_i^\top\right)^{-1}\left(\sum_{i=1}^N y_i{\bf x}_i\right)
+
+where :math:`N` is the number of training samples.
+
+A bias term :math:`b` can also be calculated as the mean error between the predictions and the true labels:
+
+.. math::
+    b = \frac{\sum_{i=1}^N(t_i-y_i)}{N}
+
+where :math:`t_i` is a true label and :math:`N` is the number of testing samples.
+
+:sgclass:`CLeastSquaresRegression` is a wrapper class for :sgclass:`CLinearRidgeRegression` with regularization coefficient :math:`\tau = 0`.
```

Comment:
sorry you got me wrong here. I meant, rename the example to be one for CLinearRidgeRegression (including the math, which is almost the same), and then mention that LeastSquares is a special case -- rather than the other way around. Get what I mean?